### PR TITLE
add dummy vars for define flags warning suppression

### PR DESCRIPTION
**Pull request by erwinmombay**
supress the warnings we get during compilation for the custom flags we pass to our `AmpCommandLineRunner` class

### CODE REVIEW

**Comment by erwinmombay** [comment]
@cramforce kind of ugly, could also do a Builder but didn't seem necessary with just 2 flags.

**Comment by erwinmombay** [comment]
will wait till https://github.com/ampproject/amphtml/pull/3887 is merged so i can recompile with my changes and dont just over write the jar

@cramforce

**Comment by cramforce** [comment]
Merged #3887

**Comment by erwinmombay** [comment]
fixed conflicts and recompiled runner.jar

**Comment by erwinmombay** [comment]
@cramforce so i tried using dummy vars, and did a full reduction (test case was 2 simple js files) and `--define` stops doing the replace when theres an `import`

**Comment by erwinmombay** [comment]
will revert this back to the old code with the java argument filter

**Comment by erwinmombay** [comment]
ping @cramforce do you know why `--define` doesn't work here? i did a full reduction of 2 simple files and the output wouldn't have the value replaced one i got into 2 modules

File: src/amp.js [hunk 1]
```diff
@@ -36,6 +36,15 @@ import {cssText} from '../build/css';
 import {maybeValidate} from './validator-integration';
 import {maybeTrackImpression} from './impression';
 
+/** @define @const {boolean} */
```
Comment (cramforce) [inline src/amp.js]: Definitely leave out the `@const`

Otherwise, no idea.

**Comment by erwinmombay** [comment]
fixed by @aghassemi unknownDefine=off work

**Pull request closed without merging**